README.md

```diff
@@ -0,0 +1,182 @@
+
+- React
+- CSS
+- Javascript ES6
+- Visual Studio Code
+- Jest
+- React Testing Library
+- ESLint
+- Stylelint
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Show list of all countries]**
+- **[Search your country easily]**
+- **[Click on each country to see more info about it]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://precious-cupcake-85c76a.netlify.app/)
+
+- [Presentation Video](https://www.loom.com/share/Contrary-React-Project-Integration-with-API-d67f63d1b9c64e10b08f7e3eaa78847d?sid=66b3a9dd-dc46-49af-a27e-dfe20eeaef82)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+Please first check this video most of your questions answeres are here :[Video link]()
+
+### Prerequisites
+
+Ensure you have:
+
+- A Web Browser such as brave or Google Chrome 🌐
+- A code editor such as Visual Studio Code 👨‍💻
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Example commands:
+
+```sh
+  cd my-folder
+  git clone https://github.com/ethred/Countries_Basic_information.git
+```
+
+### Install
+
+Install this project with:
+
+Example command:
+
+```sh
+  cd my-project
+  npm i
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+Example command:
+
+```sh
+  npm start
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+Example command:
+
+```sh
+  npm run test
+```
+
+### Deployment
+
+You can deploy this project using:
+
+Example:
+
+```sh
+  npm run deploy
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+👤 **Redwan Mohammed-Nur**
+
+- GitHub: [@ethred](https://github.com/ethred)
+- Twitter: [@redwanthr](https://twitter.com/@redwanthr)
+- LinkedIn: [Redwan Tahir](https://www.linkedin.com/in/redwan-tahir-78260733/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- **[Show more info in country details]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/M-AminAlizadeh/CountriesOfTheWorld/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project give a star to it
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank the Microverse to let me write this
+
+Creative Commons license of the design
```

Comment:
- [ ] Kindly pay attention to the details of the suggestion from the previous reviewer. We are to give credit to Nelson Sakwa the designer of the template used, not Creative Commons. It is important to give proper credit as it would be an infringement to use people's designs without acknowledgement.

  Creative Commons is just a nonprofit organization that provides a standardized way of granting permission for people to use other people's creative work. So CC is not the creator of the design used for this project.